analyzers/.sonarlint/sonaranalyzer-dotnetCSharp.ruleset

```diff
@@ -215,6 +218,7 @@
     <Rule Id="S3366" Action="Warning" />
     <Rule Id="S3376" Action="Info" />
     <Rule Id="S3397" Action="Info" />
+    <Rule Id="S3398" Action="Info" />
```

Comment:
S3398 - "private" methods called only by inner classes should be moved to those classes